README.md

```diff
@@ -440,6 +495,69 @@ Many decisions around granting equity to employees are approved by the board of
 
 🚧 This section could be expanded, and also include more legal links.
 
+### IPOs
+
+🄳 A private company becomes a public company (or “**goes public**”) in a process called an
+[**initial public offering (IPO)**](https://en.wikipedia.org/wiki/Initial_public_offering).
+Historically, only private companies with a strong track record of years of growth have
+considered themselves ready to take this significant step.
+The IPO includes a lot of regulatory costs
+[in exchange for the benefits](https://www.investopedia.com/university/ipo/ipo.asp) of
+significant capital.
+After a company goes public, investors and the general public can buy stock, and existing
+shareholders can sell their stock far more easily than when the company was private.
+
+Companies take years to IPO after being formed.
+[This Harvard report](https://corpgov.law.harvard.edu/2017/05/25/2017-ipo-report/) shows that
+the median time between a company’s founding and its IPO has been increasing;
+in **2016** it took **7.7 years**, compared to **3.1 years** in **1996**.
+
+🚧 What are the restrictions and regulations on selling stock that affect employees at
+IPO? What is a lockup period?
+
+### Sales and liquidity
+
+❗️ With private companies, it can be [very hard to know](#what-is-private-stock-worth) the
+value of equity.
+Because the value of private company stock is not determined by regular trades on public
+markets, shareholders can only make educated guesses about the likely future value, at a
+time when they will be able to sell stock.
+After all, private company stock is simply a legal agreement that entitles you to
+something of highly uncertain value, and could well be worthless in the future, or highly
+valuable, depending on the fate of the company.
+
+☝️ We’ll discuss the notion of a company officially assigning a
+[fair market value](#409a-valuations) later, but even if a company gives you a value for your
+stock for tax and accounting purposes, it doesn’t mean you can expect to sell it for that
+value!
+
+🄳 An [**acquisition**](https://www.investopedia.com/terms/a/acquisition.asp) is when one company
+(the purchaser) acquires ownership of most or all of the shares of another company (the
+company being bought).
+It’s also called a **sale** of the acquired company.
+
+🄳 A sale or IPO is often called an **exit** or **liquidity event**. Sales, dissolutions, and
+bankruptcy are all called **liquidations**.
+
+🄳 The ability to buy and sell stock is called **liquidity**. In startups and many private
+companies, it is often hard to sell stock until the company is sold or IPOs, so there is
+little or no liquidity for shareholders until those events occur.
+
+Often people wish they could sell stock in a private company, because they would prefer
```

Comment:
bit of new content here

Reply:
👍 just introducing secondary market here puts what was previously unhoused but important into context that makes it stick!